magicwand-data-generator/magicwand/magicwand_cli/cli.py

```diff
@@ -192,12 +192,12 @@ def calibrate_command(attack: str, ratio: str, verbose: str) -> None:
 
 @click.command("run")
 @click.option("--config", help="JSON file with run parameters", required=True)
-@click.option("--count", default="1", help="Number of runs", required=True, type=int)
-@click.option("--verbose", "-v", is_flag=True, help="Print more output.")
 @click.option(
```

Comment:
I moved the required options to be first so that they show up first in the `--help` docs.

Reply:
good call